lib/qb_integration/services/customer.rb

```diff
@@ -155,6 +155,9 @@ def check_param(e)
         if config.fetch("quickbooks_create_or_update") == "1"
           new_customer = create_model
           build(new_customer)
+          if config.fetch("realmId") == "123146406203559"
+            puts new_customer.inspect
```

Comment:
Was this for debugging purposes? Should it stay on the code?

Reply:
Yeah I saw this too and wasn't sure what it was for. Seems like testing for a specific client maybe?